android/KMEA/app/src/main/java/com/tavultesoft/kmea/cloud/CloudApiTypes.java

```diff
@@ -56,12 +56,20 @@ public CloudApiReturns(ApiTarget target, JSONObject jsonObject) {
      *  Keyboard download: download keyboard data package and fonts.
      */
     KeyboardData,
+    /**
+     * Keyboard package download:
```

Comment:
Looks like you meant part of line 70's comment to be here?

Reply:
kinda. I'll clarify the ApiTarget descriptions